src/commands/sfpowerkit/source/pmd.ts

```diff
@@ -16,7 +17,7 @@ core.Messages.importMessagesDirectory(__dirname);
 // or any library that is using the messages framework can also be loaded this way.
 const messages = core.Messages.loadMessages("sfpowerkit", "source_pmd");
 
-export default class Pmd extends SfdxCommand {
+export default class Pmd extends SFPowerkitCommand {
```

Comment:
The `Pmd` class needs an explicitly defined `--loglevel` flag where the default level is set to INFO, otherwise the level is WARN which stops the header in `SfpowerkitCommand` from printing.